commands/details.js

```diff
@@ -46,9 +50,23 @@ async function details (argv, arg1, arg2, arg3) {
     return
   }
 
+  const tree = await universalModuleTree(dir)
+  const list = universalModuleTree.flatten(tree)
```

Comment:
fyi this is also the return type of `ncm-analyze-tree`, if you want to reuse that instead.